src/components/Ui/Avatar.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <span>
+    <img
+      v-if="imgsrc && !error"
+      :src="imgsrc"
+      :style="{
+        width: `${parseInt(size) || 22}px`,
+        height: `${parseInt(size) || 22}px`
+      }"
+      @error="error = true"
+      class="circle border line-height-0 v-align-middle"
+    />
+    <jazzicon
+      v-else
+      :seed="seed"
```

Comment:
Is `seed` being used here? 
```suggestion
```

Reply:
We need the seed because networks and plugins don't have a unique hex string. Jazzicon `address` only takes a hex string and seed takes a positive number, so for networks I used the network ID and for plugins I used `plugin.name.charCodeAt()`to generated a random number from the name.

Reply:
I mean a number from the string, it's not random.